### PR TITLE
GTK3: patch to fix the build for MSVC x64

### PR DESCRIPTION
**Pull request by accel2k**

### CODE REVIEW

File: gtk3/build/win32/vs12/gtk3-version-paths.props [hunk 1]
```diff
@@ -54,5 +55,8 @@
     <BuildMacro Include="PythonPath">
       <Value>$(PythonPath)</Value>
     </BuildMacro>
+    <BuildMacro Include="PythonPathX64">
+      <Value>$(PythonPath)</Value>
```
Comment (nacho) [inline gtk3/build/win32/vs12/gtk3-version-paths.props]: here the value should be $(PythonPathX64)

**Comment by accel2k** [comment]
Fixed

**Comment by nacho** [comment]
Looks good, can you squash the patches?

**Comment by accel2k** [comment]
Like this?

**Comment by nacho** [comment]
Perfect thanks

**Pull request merged**